docs/api/browser-window.md

```diff
@@ -1262,6 +1262,11 @@ Sets the opacity of the window. On Linux does nothing.
 
 Returns `Number` - between 0.0 (fully transparent) and 1.0 (fully opaque)
 
+#### `win.setShape(rects)` _Windows_ _Linux_ _Experimental_
+
+* `rects` [Rectangle[]](structures/rectangle.md) - Sets a shape on the window.
```

Comment:
Can you expand this documentation a bit to explain what shapes are and why you'd want to use them?

Reply:
done